app/modules/security/security_model.php

```diff
@@ -32,38 +32,19 @@ public function __construct($serial = '')
      **/
     public function process($data)
     {
-        // Delete previous entries
-        $this->deleteWhere('serial_number=?', $this->serial_number);
-        
-        // Translate security strings to db fields
-        $translate = array(
-            'Gatekeeper: ' => 'gatekeeper',
-            'SIP: ' => 'sip');
+	require_once(APP_PATH . 'lib/CFPropertyList/CFPropertyList.php');
+	$parser = new CFPropertyList();
+	$parser->parse($data);
```

Comment:
Stole some logic from the Bluetooth module, with 39ce9739 it will parse both old (txt) and new (plist).